runtime/caches/fileSystem.ts

```diff
@@ -62,9 +62,9 @@ function createFileSystemCache(): CacheStorage {
       const fileContent = await Deno.readFile(filePath);
       return fileContent;
     } catch (err) {
-      if (err.code === "ENOENT") {
-        logger.warning(`file not found when reading from file system, path: ${FILE_SYSTEM_CACHE_DIRECTORY}/${key}`);
-      }else {
+      // Error code different for file/dir not found
+      // The file won't be found in cases where it's not cached
+      if (err.code !== "ENOENT") {
         logger.error(`error when reading from file system, ${err}`);
```

Comment:
this should be removed right? the annoying message that we mentioned

Reply:
The

`
if (err.code !== "ENOENT")
`

Takes care of the annoying message, the error will only be called in case the error is not a *not found*/*not existing* (which was the annoying error).